contrib/redhat/collectd.spec

```diff
@@ -262,6 +245,18 @@
 %define with_xmms 0
 %endif
 
+# Plugins not buildable on RHEL 8
+%if 0%{?rhel} && 0%{?rhel} >= 8
```

Comment:
%if 0%{?rhel } > 7

Reply:
that's the same thing

Reply:
but if you mean drop the first test and shorten this to `%if  0%{?rhel} >= 8` then yes, good idea :-)

Reply:
I prefer to keep it that way. It was > before and someone asked me to use >= instead. ping pong